API.md

```diff
@@ -91,6 +91,14 @@ An `HTTP 404 Not Found` status code is returned if the case with the specified I
 ]
 ```
 
+An 'HTTP 204 No Content' status code is returned is the aborted action with the specified ID has been rescheduled.
+An 'HTTP 404 Not Found' status code is returned when an invalid ID is specified.
+An 'HTTP 400 Bad Request' status code is returned when trying to rerun an action in a non aborted state.
+
+## Aborted Actions can be Rerun
+* `PUT /actions/rerun?{actionIds}` will rerun all actions for the given ID.
```

Comment:
the documentation should be `PUT /actions/rerun?actionId={actionId}` and the description should probably not say 'all actions', just 'the action specified'